scripts/load_test/chat.py

```diff
@@ -0,0 +1,26 @@
+import gradio as gr
+from time import sleep
+version, _, _ = gr.__version__.split(".")
+
+with gr.Blocks() as demo:
+    chatbot = gr.Chatbot()
+    text = gr.Textbox()
+    time = gr.Number(label="Time to Complete")
+
+    def respond(text):
+        output = ["Lorem"] * 500
+        for i in range(len(output) + 1):
+            yield [[text, " ".join(output[:i])]]
+            sleep(0.01)
+
+    if version == "3":
+        text.submit(respond, text, chatbot)
+    else:
+        text.submit(respond, text, chatbot, concurrency_limit=None)
+
+
+if __name__ == "__main__":
+    if version == "3":
+        demo.queue(concurrency_limit=None).launch(max_threads=250)
```

Comment:
This doesn't work for gradio 3.50.2 for me -  it's `concurrency_count=250`  (can't be None)

Reply:
When I made that modification, I get `OSError: [Errno 24] Too many open files` when running the 250 concurrent users case for gradio 3.x. 

What version of 3 are you using?

Reply:
I was using 3.50 as well, behind an nginx server

Reply:
Cool, I think we should make a note of that for reproducibility. But I guess this is a limitation of WS to keep in mind.